indico/core/db/sqlalchemy/review_questions.py

```diff
@@ -50,6 +50,13 @@ def event_id(cls):
             nullable=False
         )
 
+    @declared_attr
+    def question_type(cls):
+        return db.Column(
+            db.Text,
```

Comment:
Not a great idea IMO, since that'd make it much harder if we ever wanted to use a signal to return the field types (like we do in surveys, contrib fields, etc) instead of supporting only a hardcoded list.

Reply:
Ah, you mean having it extended by plugins? Good point.